src/client/login/Login.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { browserHistory } from 'react-router';
+import styles from './Login.css';
+
+export default class Login extends Component {
+  constructor(props){
+    super(props);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+  onSubmit(e){
+    e.preventDefault();
+    var username = document.getElementById("inputfield").value;
```

Comment:
You shouldn't need to do this in react. The pattern to use would be to have an `onChange` event on the input and use component state to save the value. There's details here https://facebook.github.io/react/docs/forms.html, but gimme a shout if you want help